src/User32.Desktop/User32.cs

```diff
@@ -493,6 +504,68 @@ public static partial class User32
+        /// </summary>
+        /// <param name="nInputs">The number of structures in the <paramref name="pInputs" /> array.</param>
+        /// <param name="pInputs">An array of  structures. Each structure represents an event to be inserted into the keyboard or mouse input stream.</param>
+        /// <param name="cbSize">The size, in bytes, of an <see cref="INPUT" /> structure. If cbSize is not the size of an <see cref="INPUT" /> structure, the function fails.</param>
+        /// <returns>
+        /// The function returns the number of events that it successfully inserted into the keyboard or mouse input stream.
+        /// If the function returns zero, the input was already blocked by another thread. To get extended error information, call GetLastError.
+        /// </returns>
+        /// <remarks>
+        /// This function is subject to UIPI. Applications are permitted to inject input only into applications that are at an equal or lesser integrity level.
+        /// This function fails when it is blocked by UIPI. Note that neither GetLastError nor the return value will indicate the failure was caused by UIPI blocking.
+        /// </remarks>
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern unsafe uint SendInput(
+            uint nInputs,
+            [Friendly(FriendlyFlags.Array), In] INPUT* pInputs,
+            int cbSize);
+
+        /// <summary>
+        /// Waits until the specified process has finished processing its initial input and is waiting for user input with no input pending, or until the time-out interval has elapsed.
+        /// </summary>
+        /// <param name="hProcess">A handle to the process. If this process is a console application or does not have a message queue, WaitForInputIdle returns immediately.</param>
+        /// <param name="dwMilliseconds">The time-out interval, in milliseconds. If dwMilliseconds is INFINITE, the function does not return until the process is idle.</param>
+        /// <returns>0 if the wait was satisfied successfully., WAIT_TIMEOUT if the wait was terminated because the time-out interval elapsed, and WAIT_FAILED if an error occurred.</returns>
+        /// <remarks>Raymond Chen has a series of articles that give a bit more depth to how this function was intended to be used.
+        /// <a href="http://blogs.msdn.com/b/oldnewthing/archive/2010/03/25/9984720.aspx">Here</a> and <a href="http://blogs.msdn.com/b/oldnewthing/archive/2010/03/26/9985422.aspx">here</a>.
+        /// The jist of it is that this function should have been really called WaitForProcessStartupComplete, as this is all it does.</remarks>
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern uint WaitForInputIdle(IntPtr hProcess, uint dwMilliseconds);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern short GetAsyncKeyState(int vKey);
```

Comment:
I don't think the int overloads are really necessary, the list of virtual keys is pretty well defined and the end-user can still cast to the enum type if needed.